src/database/database.py

```diff
@@ -1,46 +1,136 @@
 import datetime
 
-from pymongo import MongoClient
 
+from typing import Dict
+from pymongo import MongoClient
 from utilities.config_loader import load_config
 
 
-def _create_default_document(id):
+def _create_default_document(id: int) -> Dict:
+    """Gives a database entity an id and a timestamp
+    
+    Parameters
+    ----------
+    id
+        id of the entity to be created
+        
+    Returns
+    -------
+    Dict
+        containing id and timestamp
+    """
+
     return {
         "song_id": id,
         "last_updated": datetime.datetime.utcnow(),
     }
+ 
+def _augment_document(doc1: dict, doc2: dict) -> Dict:
+    """Combines parameters into a larger dictionary
+    
+    Parameters
+    ----------
+    doc1
+        first dictionary
+    doc2
+        second dictionary
+        
+    Returns
+    -------
+    Dict
+        dictionary combining two dictionaries
+    """
 
-
-def _augment_document(doc1, doc2):
     return {**doc1, **doc2}
 
 
+# Generic class for making functions implementable
+# for lower level classes of music analysis
 class Database:
```

Comment:
You should also add some documentation to the class itself. Something like this:
```
    """
    A class used to represent an Animal

    ...

    Attributes
    ----------
    says_str : str
        a formatted string to print out what the animal says
    name : str
        the name of the animal
    sound : str
        the sound that the animal makes
    num_legs : int
        the number of legs the animal has (default 4)

    Methods
    -------
    says(sound=None)
        Prints the animals name and what sound it makes
    """
```